packages/react/src/breadcrumb/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Breadcrumb, Breadcrumbs } from '../breadcrumb';
+
+const meta: Meta<typeof Breadcrumbs> = {
+  title: 'Breadcrumbs',
+  component: Breadcrumbs,
+
+  render: () => {
+    return (
+      <Breadcrumbs>
+        <Breadcrumb href="/">Min side</Breadcrumb>
+        <Breadcrumb href="/">Instillinger</Breadcrumb>
```

Comment:
Tror alle eksemplene i storybooken burde skrives om til å bruke `#` i stedet for `/` som href. Trykker du på en lenke nå tas du ut av storyen.